src/app/dataset-view/dataset.service.ts

```diff
@@ -52,6 +53,17 @@ export class DatasetService {
         this.datasetChanges$.next(searchDatasetInfo);
     }
 
+    private metadataBlockChanges$: Subject<MetadataBlockFragment> =
```

Comment:
Let's move these extensions to a separate service, specific to `MetadataBlock` module

Reply:
Moved to new block.service in MetadataBlock module